src/main/java/com/group12/ciserver/database/DatabaseWrapper.java

```diff
@@ -0,0 +1,193 @@
+                }
+            }
+
+            return -1;
+        } catch (SQLException e) {
+            System.err.println(e.getMessage());
+            return -1;
+        }
+    }
+
+    public boolean removeBuild(long uid) {
+        String sql = "delete from builds where uid = ?";
+        try {
+            PreparedStatement pstmt = connection.prepareStatement(sql);
+            pstmt.setLong(1, uid);
+
+            int affectedRows = pstmt.executeUpdate();
+
+            return affectedRows == 1;
+        } catch (SQLException e) {
+            System.err.println(e.getMessage());
+            return false;
+        }
+    }
+
+    public boolean editBuildContent(long uid, String content) {
+        String sql = "update builds set content = ? where uid = ?";
+
+        try {
+            PreparedStatement pstmt = connection.prepareStatement(sql);
+            pstmt.setString(1, content);
+            pstmt.setLong(2, uid);
+
+            int affectedRows = pstmt.executeUpdate();
+
+            return affectedRows == 1;
+        } catch (SQLException e) {
+            System.err.println(e.getMessage());
+            return false;
+        }
+    }
+
+    public BuildInfo getBuildInfo(long uid) {
+        String sql = "select * from builds where uid = ?";
+
+        try {
+            PreparedStatement pstmt = connection.prepareStatement(sql);
+            pstmt.setLong(1, uid);
+            ResultSet rs = pstmt.executeQuery();
+
+            if(rs.next())
+            {
+                String commit_hash = rs.getString(2);
+                String content = rs.getString(3);
+                String ts = rs.getString(4);
+                Date date = sqliteTimestampToDate(ts);
+                return new BuildInfo(uid, commit_hash, content, date);
+            }
+
+            return null;
+        } catch (SQLException e) {
+            System.err.println(e.getMessage());
+            return null;
+        }
+    }
+
+    public ArrayList<BuildInfo> getAllBuilds() {
+        String sql = "select * from builds";
+        ArrayList<BuildInfo> builds = new ArrayList<>();
+        try {
+            PreparedStatement pstmt = connection.prepareStatement(sql);
+            ResultSet rs = pstmt.executeQuery();
+
+            while(rs.next())
+            {
+                long uid = rs.getLong(1);
+                String commit_hash = rs.getString(2);
+                String content = rs.getString(3);
+                String ts = rs.getString(4);
+                Date date = sqliteTimestampToDate(ts);
+                builds.add(new BuildInfo(uid, commit_hash, content, date));
+            }
+
+            return builds;
+        } catch (SQLException e) {
+            System.err.println(e.getMessage());
+            return null;
+        }
+    }
+
+    public static Date sqliteTimestampToDate(String ts)
+    {
+        try {
+            DateFormat dateFormat = new SimpleDateFormat("yyyy-MM-dd HH:mm:ss");
```

Comment:
I would suggest using `DateTimeFormatter.ISO_OFFSET_DATE_TIME` format. https://docs.oracle.com/javase/8/docs/api/java/time/format/DateTimeFormatter.html#ISO_OFFSET_DATE_TIME

It is an ISO standard for datetimes.